src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -17,7 +17,7 @@
      *   <br>examples:<pre>
      *       containsWordIgnoreCase("ABc def", "abc") == true
      *       containsWordIgnoreCase("ABc def", "DEF") == true
-     *       containsWordIgnoreCase("ABc def", "AB") == false //not a full word match
+     *       containsWordIgnoreCase("ABc def", "AB") == true //for a word that contains the substring
```

Comment:
Is this change for implementation of substring branch instead of this mixed-case branch?

Reply:
Nope